src/components/pages/devices/flyouts/deviceDetails/deviceDetails.js

```diff
@@ -144,37 +144,34 @@ export class DeviceDetails extends Component {
                 }
               </Accordion>
 
-              <Accordion title={t('devices.details.methods.title')} description={t('devices.details.methods.description')}>
+              <Accordion title={t('devices.flyouts.details.methods.title')} description={t('devices.flyouts.details.methods.description')}>
                 {
-                  (methods.length === 0) &&
-                  t('devices.details.methods.noneExist')
-                }
-                {
-                  (methods.length > 0) &&
-                  <Grid>
-                    {
-                      methods.map((methodName, idx) =>
-                        <Row key={idx}>
-                          <Cell>{methodName}</Cell>
-                        </Row>
-                      )
-                    }
-                  </Grid>
+                  (methods.length === 0)
+                    ? t('devices.flyouts.details.methods.noneExist')
+                    : <Grid>
+                      {
+                        methods.map((methodName, idx) =>
+                          <Row key={idx}>
+                            <Cell>{methodName}</Cell>
+                          </Row>
+                        )
+                      }
+                    </Grid>
```

Comment:
This needs to be aligned with the grid above.

Reply:
Formatting the file will just move it back to here.